meta-dts-distro/recipes-dts/reports/dasharo-hcl-report/dasharo-hcl-report

```diff
@@ -184,11 +190,9 @@ if [ $BOARD_HAS_FD_REGION -eq 0 ]; then
     # GBE region is present and not locked, read it as well
     FLASHROM_ADD_OPT_READ+=" -i gbe"
   fi
-else
-    # No descriptor, probably safe to read everything
-    FLASHROM_ADD_OPT_READ=" "
 fi
-$FLASHROM -V -p internal:laptop=force_I_want_a_brick ${FLASH_CHIP_SELECT} -r logs/rom.bin ${FLASHROM_ADD_OPT_READ} > logs/flashrom_read.log 2> logs/flashrom_read.err.log
```

Comment:
We remove the `internal:laptop=force_I_want_a_brick` option here, but  do we add it somewhere else to the platforms which may need it?

What are the tests results of this code, which scenarios/platforms have been tested?

Reply:
Most likely, this option is required on some laptops. Is this only for HCL report? How about keeping the `internal:laptop=force_I_want_a_brick` option for APU as well? It should not harm anything, or does it? @PLangowski @TomaszAIR ?

Reply:
@macpijan this was only for HCL report. We added PROGRAMMER_BIOS var from board_config function which is now called at the beginning of the report.

@PLangowski did you successfully dump firmware with this option on APU? If yes I think we can keep it here. Or maybe better move to board_config func so we would have one place where initial options are set.

Reply:
@TomaszAIR Yes, the firmware is dumped to `logs/`, which is then archived. The shasum of the dumped fw is the same as of the original. So what should we do here?

Reply:
@TomaszAIR Yes, I just verified that using the `internal:laptop=force_I_want_a_brick` option dumps the fw correctly. I verified the dumped file's checksum with the original's and it's the same. What should we do with it then?

Reply:
@macpijan @PLangowski if dump worked with `internal:laptop=force_I_want_a_brick` then I would leave it as it was. Right now I am not sure which exactly platforms may need it so can't tell to which one this should be added.

Reply:
Fixed.